docs/tutorials/intro.md

```diff
@@ -0,0 +1,177 @@
+# Getting Started with JupyterGIS
+
+Welcome to the first tutorial in the JupyterGIS series! This guide will introduce you to the basics of JupyterGIS and help you set up your environment, create your first map, and understand its key features.
+
+## Objectives
+
+By the end of this tutorial, you will:
+
+1. Understand what JupyterGIS is and its key capabilities.
+2. Set up the required tools and libraries.
+3. Create a simple map and add layers.
+4. Explore basic geospatial analysis using JupyterGIS.
+
+## Introduction to JupyterGIS
+
+JupyterGIS integrates geospatial analysis with the interactivity of Jupyter Notebooks, making it easy to:
+
+- Visualize and create interactive maps.
+- Process and analyze geospatial data.
+- Share and collaborate on geospatial workflows.
+
+JupyterGIS is designed for researchers, data scientists, and GIS professionals aiming to combine Python’s analytical power with intuitive geospatial visualizations.
+
+## An Overview of the Interface
+
+We will explore the JupyterGIS user interface to help you become familiar with its menus, toolbars, map canvas, and layers list, which make up the core structure of the interface.
+
+**The objective of this lesson is to understand the fundamentals of the JupyterGIS user interface.**
+
+![JupyterGIS User Interface explained](../assets/images/jupyterGIS_interface.png)
+
+The elements shown in the figure above are:
+
+1. The application Launcher helps you to select which application you want to start e.g. a Notebook, a Console, and another application such as a Terminal or open a GIS file (which can be either a JupyterGIS file or a QGIS file).
+2. GIS Layers List / Browser Panel
+3. Left Sidebar which contains a file browser, a list of tabs in the main work and of running kernels and terminals, the command palette, the table of contents, the extension manager, and the JupyterGIS extension which allows you to see the GIS layers list.
+4. The GIS object properties, annotations and filters of a selected GIS layer.
+5. The Jupyter toolbar menu which you will use with Jupyter Notebooks.
+6. The Log console (which you can you for debugging).
+7. Right Sidebar showing the property inspector (active in notebooks), the debugger, and GIS object properties, annotation and filters.
+
+## Adding your first layers
+
+We will start the JupyterGIS application and create a basic map.
+
+**The goal for this lesson: To get started with an example map.**
+
+:::{admonition} Get JupyterGIS
```

Comment:
Since you're using colon fences here and we're using myst-parser, you'll have to add the `colon_fence` extension to our `conf.py`. See https://github.com/jupyterlite/xeus/blob/main/docs/conf.py#L9 for an example of this.

Reply:
Thank you!